Sources/ComposableCoreLocation/Interface.swift

```diff
@@ -395,6 +399,10 @@ public struct LocationManager {
 
   public func location(id: AnyHashable) -> Location? { self.location(id) }
 
+  public func accuracyAuthorization(id: AnyHashable) -> AccuracyAuthorization? {
```

Comment:
One small change. Let's guard the client from calling code that is unavailable:
```suggestion
  @available(iOS 14.0, tvOS 14.0, watchOS 7.0, macOS 11.0, macCatalyst 14.0, *)
  public func accuracyAuthorization(id: AnyHashable) -> AccuracyAuthorization? {
```